engine/glfw/lib/cocoa/cocoa_window.m

```diff
@@ -1178,3 +1178,11 @@ void _glfwPlatformSetViewType(int view_type)
 GLFWAPI void glfwAccelerometerEnable()
 {
 }
+
+void _glfwPlatformSetWindowBackgroundColor(int color)
```

Comment:
I would use an "unsigned int" to assure there's no strange things happening with the signed bit.
It might work well, since we currently don't use alpha. (why not?)